src/attributes/components/AttributeProperties/AttributeProperties.tsx

```diff
@@ -77,39 +80,41 @@ const AttributeProperties: React.FC<AttributePropertiesProps> = ({
           />
         </Typography>
         <Hr />
-        {data.type === AttributeTypeEnum.PRODUCT_TYPE && (
-          <>
-            <ControlledCheckbox
-              name={"filterableInStorefront" as keyof FormData}
-              label={intl.formatMessage({
-                defaultMessage: "Use in Faceted Navigation",
-                description: "attribute is filterable in storefront"
-              })}
-              checked={data.filterableInStorefront}
-              onChange={onChange}
-              disabled={disabled}
-            />
-            <FormSpacer />
-          </>
-        )}
-        {data.filterableInStorefront &&
+        {data.inputType !== AttributeInputTypeEnum.FILE &&
           data.type === AttributeTypeEnum.PRODUCT_TYPE && (
-            <TextField
-              disabled={disabled}
-              error={!!formErrors.storefrontSearchPosition}
-              fullWidth
-              helperText={getAttributeErrorMessage(
-                formErrors.storefrontSearchPosition,
-                intl
+            <>
+              <ControlledCheckbox
+                name={"filterableInStorefront" as keyof FormData}
+                label={intl.formatMessage({
+                  defaultMessage: "Use in Faceted Navigation",
+                  description: "attribute is filterable in storefront"
+                })}
+                checked={data.filterableInStorefront}
+                onChange={onChange}
+                disabled={disabled}
+              />
+              <FormSpacer />
```

Comment:
If the condition below (`filterableInStorefront`) doesn't pass we'll have 2 FormSpacer components on top of each other, is this the intended behaviour?